bps/bps_ApTemplate.yaml

```diff
@@ -0,0 +1,47 @@
+# UPDATE THIS to the path to the pipeline to run
+pipelineYaml: '${AP_PIPE_DIR}/pipelines/ApTemplate.yaml'
+
+# Format for job names and job output filenames
+templateDataId: '{tract}_{patch}_{band}_{visit}_{exposure}_{detector}'
+
+# Just names, but UPDATE THIS AND KEEP THEM SHORT
+project: ApTemplate
+campaign: DM-xxxxx
+
+submitPath: ${PWD}/bps/{outCollection}
+computeSite: ncsapool
+# Memory allocated for each quantum, in MBs
```

Comment:
Do you know the maximum that can be requested? If so, that would be good to include.

Reply:
I don't know, and therefore haven't included it 🙃 in practice, 2000-4000 is enough for most jobs, and bumping it up to 12000 or so is enough to get the occasional held job through.